simul/platform/deterlab/users/users.go

```diff
@@ -171,3 +162,10 @@ func deterFromConfig(name ...string) *platform.Deterlab {
 	dbg.SetDebugVisible(d.Debug)
 	return d
 }
+
+// Runs a command on the remote host and outputs eventual error
```

Comment:
Runs a command on the remote host and outputs eventual error (if debug level >= 3)